fiftyone/utils/sharp.py

```
@@ -0,0 +1,223 @@
+
+    def _export_gaussians(self, gaussians, f_px, height, width):
+        """Export gaussians to a PLY file and return a Classification label."""
+        self._ensure_output_dir()
+        splat_id = uuid.uuid4().hex[:12]
+        splat_path = os.path.join(
+            self._output_dir, "splat_%s.ply" % splat_id
+        )
+        sharp_utils.save_ply(gaussians, f_px, (height, width), splat_path)
+        return fol.Classification(label="3d_gaussians", splat_path=splat_path)
+
+    def _predict_all(self, imgs):
```

Ruff (0.15.2)

[warning] Missing return type annotation for private function `_predict_all`

(ANN202)